iset.mm

```diff
@@ -97573,6 +97724,256 @@ Knuth for iterated exponentiation (_Science_ 194, 1235-1242, 1976) and
+       Kingdon, 21-Feb-2022.) $)
+    sizefz1 $p |- ( N e. NN0 -> ( size ` ( 1 ... N ) ) = N ) $=
+      ( vx vy cn0 wcel c1 cfz co csize cfv cz cv caddc cmpt cc0 cfrec wceq wf1o
+      com syl2anc ccnv cen wbr cuz wf eqid frec2uzf1od f1ocnv f1of 3syl elnn0uz
+      biimpi ffvelrnd frecfzennn ensymd sizeennn wa oveq1 cbvmptv freceq1 ax-mp
+      0zd fveq1i f1ocnvfv2 syl5eqr eqtrd ) ADEZFAGHZIJZABKBLZFMHZNZOPZUAZJZCKCL
+      ZFMHZNZOPZJZAVGVOSEVOVHUBUCVIVTQVGOUDJZSAVNVGSWAVMRZWASVNRWASVNUEVGBOVMVG
+      VBVMUFZUGZSWAVMUHWASVNUIUJVGAWAEZAUKULZUMVGVHVOBVMAWCUNUOCVHVOUPTVGWBWEVT
+      AQWDWFWBWEUQVTVOVMJAVOVMVSVLVRQVMVSQBCKVKVQVJVPFMURUSOVLVRUTVAVCSWAAVMVDV
+      ETVF $.
+
+    $( Two finite sets have the same number of elements iff they are
+       equinumerous.  (Contributed by Jim Kingdon, 21-Feb-2022.) $)
+    sizeen $p |- ( ( A e. Fin /\ B e. Fin ) ->
+                   ( ( size ` A ) = ( size ` B ) <-> A ~~ B ) ) $=
+      ( vn vm vx cfn wcel wa cv cen wbr csize cfv wceq com wrex syl2anc syl cc0
+      wb isfi biimpi adantr ad2antlr simplrl simprl nneneq simplrr enen1 simprr
+      enen2 bitrd cz c1 caddc co cmpt cfrec ensymd sizeennn eqeq12d cuz wf1 0zd
+      wf1o eqid frec2uzf1od f1of1 f1fveq syl12anc 3bitr4rd rexlimddv ) AFGZBFGZ
+      HZACIZJKZALMZBLMZNZABJKZTZCOVMVQCOPZVNVMWCCAUAUBUCVOVPOGZVQHZHZBDIZJKZWBD
+      OVNWHDOPZVMWEVNWIDBUAUBUDWFWGOGZWHHZHZVPWGJKZVPWGNZWAVTWLWDWJWMWNTVOWDVQW
+      KUEZWFWJWHUFZVPWGUGQWLWAVPBJKZWMWLVQWAWQTVOWDVQWKUHZAVPBUIRWLWHWQWMTWFWJW
+      HUJZBWGVPUKRULWLVTVPEUMEIUNUOUPUQSURZMZWGWTMZNZWNWLVRXAVSXBWLWDVPAJKVRXAN
+      WOWLAVPWRUSEAVPUTQWLWJWGBJKVSXBNWPWLBWGWSUSEBWGUTQVAWLOSVBMZWTVCZWDWJXCWN
+      TWLOXDWTVEXEWLESWTWLVDWTVFVGOXDWTVHRWOWPOXDVPWGWTVIVJULVKVLVL $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.
+    $( The size of two finite sets is equal if and only if there is a bijection
+       mapping one of the sets onto the other.  (Contributed by Alexander van
+       der Vekens, 17-Dec-2017.) $)
+    sizeeqf1o $p |- ( ( A e. Fin /\ B e. Fin )
+        -> ( ( size ` A ) = ( size ` B ) <-> E. f f : A -1-1-onto-> B ) ) $=
+      ( cfn wcel wa csize cfv wceq cen wbr cv wf1o wex sizeen bren syl6bb ) ADE
+      BDEFAGHBGHIABJKABCLMCNABOABCPQ $.
+
+    $( There is no bijection between a finite set and an infinite set.  By
+       ~ infnfi the theorem would also hold if "infinite" were expressed as
+       ` _om ~<_ B ` .  (Contributed by Alexander van der Vekens,
+       25-Dec-2017.) $)
+    fiinfnf1o $p |- ( ( A e. Fin /\ -. B e. Fin )
+                      -> -. E. f f : A -1-1-onto-> B ) $=
+      ( cfn wcel cv wf1o wex f1ofi ex exlimdv con3dimp ) ADEZABCFZGZCHBDEZMOPCM
+      OPABNIJKL $.
+  $}
+
+  $( The codomain of an onto function is a set if its domain is a set.
+     (Contributed by AV, 4-May-2021.) $)
+  focdmex $p |- ( ( A e. V /\ F : A -onto-> B ) -> B e. _V ) $=
+    ( wcel wfo wa crn cvv wf fof anim2i ancomd rnexg 3syl wb forn eleq1d adantl
```

Comment:
should probably go earlier since not about size.

Reply:
I don't object to moving it although I'd move it in both set.mm and iset.mm if so.